src/Client/Magic.fs

```diff
@@ -312,17 +312,15 @@ let query (model : Model) msgs =
         stringQuery
         |> AsyncRx.map Shared.Msg.LetterStringChanged
 
-      let xs =
-        stringQuery
-        |> AsyncRx.startWith [model.LetterString |> extractedLetterString]
-        |> AsyncRx.flatMapLatest (fun letters ->
-            letterStream letters)
-        |> AsyncRx.map Shared.Msg.Letter
-        |> AsyncRx.merge letterStringQuery
-        |> server
-        |> AsyncRx.map RemoteMsg
-
-      Subscribe (xs, "_remote")
+      stringQuery
+      |> AsyncRx.startWith [model.LetterString |> extractedLetterString]
+      |> AsyncRx.flatMapLatest (fun letters ->
+          letterStream letters)
+      |> AsyncRx.map Shared.Msg.Letter
+      |> AsyncRx.merge letterStringQuery
+      |> server
+      |> AsyncRx.map RemoteMsg
+      |> AsyncRx.asQuery "_remote"
```

Comment:
I like this. Maybe we could make this the default way of working with streams that are constructed with pipelines? We could remove most of the Query (xs, "name") lines and the intermediate identifiers

Reply:
Yes, that was my idea trying to make it more composable and wondering if the name should be first or last to the DU case. By making an "operator" `asQuery` I can curry the source so I don't need to bind it to a name first. 

Also decided to call it a `Query` and not `Subscribe` since `Subscribe` can be easy to confuse with `SubscribeAsync`, but we don`t know who (the observer downstream) is subscribing. We're just giving it a name to identify it. But naming is hard :) Anyways, this is what is released now, but we can always refactor things for the next version when we have more experience with using it.

Reply:
Great. Personally I am not totally happy with Query either. But I will open a ticket in Reaction about this